include/sdf/Plugin.hh

```diff
@@ -188,6 +227,16 @@ namespace sdf
       return _in;
     }
 
+    /// \brief Initializer function to help Plugin constructors.
+    /// \param[out] _errors Vector of errors.
+    /// \param[in] _filename Filename of the shared library associated with
+    /// this plugin.
+    /// \param[in] _name The name of the plugin.
+    /// \param[in] _xmlContent Optional XML content that will be stored in
+    /// this plugin.
+    private: void Init(sdf::Errors &_errors, const std::string &_filename,
+                    const std::string &_name, const std::string &_xmlContent);
+
     /// \brief Private data pointer.
     std::unique_ptr<sdf::PluginPrivate> dataPtr;
```

Comment:
Apparently, the `dataPtr` was public before (yikes!), and now that there's a `private:` member right above, it's changing `dataPtr` to private as well, breaking ABI.

Reply:
Damn, I made it public then. I will follow up with a PR to main and make sure `dataPtr`s are not public there.